### PR TITLE
docs: description++

### PR DESCRIPTION
**Pull request by mgrudzinska**

### CODE REVIEW

File: inc/thorvg.h [hunk 1]
```diff
@@ -1286,7 +1286,7 @@ class TVG_API Picture final : public Paint
      * when the @p copy has @c false. This means that loading the same data again will not result in duplicate operations
      * for the sharable @p data. Instead, ThorVG will reuse the previously loaded picture data.
      *
-     * @param[in] data A pointer to a memory location where the content of the picture file is stored.
+     * @param[in] data A pointer to a memory location where the content of the picture file is stored, null-terminated for non-binary data.
```
Comment (hermet) [inline inc/thorvg.h:1289]: Let's carefully consider this. It likely applies only when copy == false.

Comment (mgrudzinska) [reply inc/thorvg.h:1289]: in lottie for now we need that in both cases

Comment (hermet) [reply inc/thorvg.h:1289]: @mgrudzinska Yes. I know that. just considering add `\0` thorvg inside in case of copy == true.

File: inc/thorvg.h [hunk 2]
```diff
@@ -1286,7 +1286,7 @@ class TVG_API Picture final : public Paint
      * when the @p copy has @c false. This means that loading the same data again will not result in duplicate operations
      * for the sharable @p data. Instead, ThorVG will reuse the previously loaded picture data.
      *
-     * @param[in] data A pointer to a memory location where the content of the picture file is stored.
+     * @param[in] data A pointer to a memory location where the content of the picture file is stored (null-terminated for non-binary data if @p copy is @c false).
```
Comment (hermet) [inline inc/thorvg.h:1289]: I think we can clarify the sentence more. `A null-terminated string is expected for non-binary data if @p copy is @c false`

**Review by hermet — APPROVED** [review]

**Pull request merged**